### DISCUSSION TITLE
Disable Lazy Load Poster Fade

### DISCUSSION THREAD

**Discussion by jona-ha** [google/model-viewer#1993] [Q&A] [answered]
Dear modelviewer community, 👋

in my current project I use the modelviewer with transparent background, lazy load on interaction and a poster. I followed this [example](https://modelviewer.dev/examples/loading/index.html#customizeLoad).

The problem: the modelviewer in its unloaded state is not black. I think it is a feature to see loadable modelviewers and to smooth out the loading experience: The moment you click on the poster or button the grey poster fades to its original color (black bg) and to the 3D model itself.

Is there any possibility to deactivate the fade/ grey overlay?

![grafik](https://user-images.githubusercontent.com/17043348/106492609-68350f80-64b8-11eb-9110-976ddf7642cb.png)
fade
![grafik](https://user-images.githubusercontent.com/17043348/106492935-cd890080-64b8-11eb-9ad1-1a4867f733fa.png)
loaded
![grafik](https://user-images.githubusercontent.com/17043348/106493176-16d95000-64b9-11eb-92a7-1837ff2eea1f.png)

- Jonathan

**Comment by elalish** [comment, marked as answer]
Ideally that should be fixable with CSS: https://modelviewer.dev/docs/index.html#entrydocs-loading-css-progressMask, though I'm concerned there may be a bug there. In any case, if you put in a custom progress bar in the slot, it will replace all of that default stuff. You can find an example with our newly updated [editor](https://modelviewer.dev/editor/) output.

**Reply by jona-ha** [reply]
Wow, thank you so much!! 👍 

here is what I did:

```
<modelviewer ...blabla... >
<div class="progress-bar" slot="progress-bar"></div>
</modelviewer>
```
and inside my style.css I added
```
.progress-bar {
  display: none;
  visibility: hidden;
}
```
